.golangci.yaml

```diff
@@ -37,10 +37,10 @@ linters:
     - 'rowserrcheck'
     - 'staticcheck'
     - 'stylecheck'
-    - 'tenv'
```

Comment:
This was marked as deprecated in favor of `usetesting` when I ran `golangci-lint run` locally.